localenv/mock-account-servicing-entity/app/lib/webhooks.server.ts

```diff
@@ -6,6 +6,7 @@ import { apolloClient } from './apolloClient'
 import { v4 as uuid } from 'uuid'
 
 export enum EventType {
+  IncomingPaymentCreated = 'incoming_payment.created',
```

Comment:
we need to also add a handler in `mock-account-servicing-entity/app/routes/webhooks.ts` otherwise we return 400 from the MASE and the webhook keeps retrying

Reply:
Indeed. Thanks!